offscreen-canvas/the-offscreen-canvas/2d.canvas.readonly.html

```diff
@@ -16,9 +16,10 @@
 var offscreenCanvas = new OffscreenCanvas(100, 50);
 var ctx = offscreenCanvas.getContext('2d');
 
+var offscreenCanvas2 = new OffscreenCanvas(100, 50);
```

Comment:
Why doe this test (and several other generated tests) create two offscreen canvases when only one is needed.